### PR TITLE
feat(services): improve service link popover behavior

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-906

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28102cd2-684e-4ecb-a4f7-2f37f653b027 | [my-helm-env](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/28102cd2-684e-4ecb-a4f7-2f37f653b027/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Don’t Sacrifice Readability, you are going to debug it in 6 months

File: libs/domains/services/feature/src/lib/service-links-popover/service-links-popover.tsx [hunk 1]
```diff
@@ -48,8 +49,10 @@ export function ServiceLinksPopover({
   }
 
   return (
-    <Popover.Root>
-      <Popover.Trigger className={filteredLinks.length === 0 ? 'hidden' : ''}>{children}</Popover.Trigger>
+    <Popover.Root modal={true}>
+      <Tooltip content="Links">
```
Comment (RemiBonnet) [inline libs/domains/services/feature/src/lib/service-links-popover/service-links-popover.tsx:53]: is it weird to have a tooltip `links` in hover when it's already written `links` on the btn?

Comment (ctjhoa) [reply libs/domains/services/feature/src/lib/service-links-popover/service-links-popover.tsx]: Fixed 
https://github.com/Qovery/console/pull/1132/commits/9985e03a0138fb0a09c38377004de6acd8906971

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**